packages/dapp/src/styles.ts

```diff
@@ -57,6 +57,41 @@ export const responsive = {
   xxl: '1400px',
 };
 
+export const theme = {
```

Comment:
do we still use these style configurations? If not this file must be removed.

Reply:
nope , deleted

Reply:
if you delete it locally it does not mean that file is deleted in the repository. )))
please delete in using the command `git rm`